src/lib/Components/Inbox/Conversations/__tests__/Conversations-tests.tsx

```diff
@@ -10,6 +10,11 @@ it("looks correct when rendered", () => {
   expect(tree).toMatchSnapshot()
 })
 
+it("doesn't render the header when there are no messages", () => {
+  const tree = renderer.create(<Conversations me={mePropsEmpty} relay={{ hasMore: jest.fn() } as any} />).toJSON()
+  expect(tree).toMatchSnapshot()
+})
```

Comment:
TBH, this test doesn't _actually_ test that, it just verifies the JSON - would recommend adding a check for the lack of that specifically - you can see some examples here: https://github.com/artsy/emission/blob/master/src/lib/Components/Consignments/Screens/__tests__/Confirmation-tests.tsx#L87-L108

Reply:
ah ok, will push after dinner 👌